fabfile.py

```diff
@@ -14,8 +14,7 @@
 ROOT = os.path.abspath(os.path.dirname(__file__))
 
 os.environ['DJANGO_SETTINGS_MODULE'] = '%s-project.settings' % NAME
-os.environ['PYTHONPATH'] = os.pathsep.join([ROOT,
-                                            os.path.join(ROOT, 'examples')])
+os.environ['PYTHONPATH'] = os.pathsep.join([ROOT, os.path.join(ROOT, 'examples')])
```

Comment:
as not-the-owner-anymore, I don't have a strong opinion, but style guide changes are usually not changes that should be made lightly in the middle of another PR. that needs to change the linter rules, and should be discussed separately